### ISSUE TITLE
Language granting REs no longer working

### ISSUE THREAD

**Issue by surged20** [velara/MercenaryMarketVolume1#36]
Since system 5.11.5, NPC languages are stored in `system.details.languages` so the current language granting REs attached to ancestory items are not working and kick out a validation error from the schema checker.

As an example, I fixed the dwarven ancestry activeeffect-like RE on one of my NPCs as follows:
```
{"key":"ActiveEffectLike","mode":"add","path":"system.details.languages.value","value":"dwarven"}
```

**Comment by velara** [comment]
Thanks for the heads up, and tracking down a solution! Sounds like it should be a pretty quick fix. 

I’ll update the ancestry templates for the next release.

**Comment by surged20** [comment]
Thanks for the fantastic module and quick response.

**Comment by velara** [comment]
Closed in 3.0.6
